docs/monitoring/spm-faq.md

```diff
@@ -656,7 +678,7 @@ spm-receiver.sematext.com 443 port
 In case you see some other result:
 
   - if your server requires proxy to access the internet, you can define
-    its settings in `/opt/spm/properties/agent.properties`. After
+    its settings using `/opt/spm/bin/setup-env` command. After
     that restart SPM agent.
```

Comment:
SPM agent, Java agent, SPM client. :)
We need to decide on which name we'll use.
SPM client is really referring to spm-client package, which really has all our agents.
SPM Monitor is the old name.
Java agent is a "new name", but confusing because Java is just the implementation language.

Public name for STA could be Sematext Infra Agent
Public name for AAs could be... Sematext App Agent? (also not ideal, because this agent may not really be used for monitoring one's _own_ application, although now that this agent is modular that may actually happen?)

Reply:
Aha, I see some agent naming changes in other commits...